test/integration/test-helper.coffee

```diff
@@ -0,0 +1,39 @@
+_ = require 'underscore'
+Promise = require 'bluebird'
+
+module.exports.unpublishProduct = (logger, client, product) ->
```

Comment:
Generally in Node, there should only be one `module.exports` if you want to export several methods you just use `exports.*`

e.g
```
exports.unpublishProduct
```